Directory.Packages.props

```diff
@@ -2,8 +2,12 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <MicrosoftBuildPackageVersion>17.6.3</MicrosoftBuildPackageVersion>
+
+    <DataflowPackageVersion Condition="'$(TargetFramework)' != 'net46'">6.0.0</DataflowPackageVersion>
+    <DataflowPackageVersion Condition="'$(TargetFramework)' == 'net46'">4.11.1</DataflowPackageVersion>
```

Comment:
This won't work for all projects so it could cause issues. The condition needs to be on the Item below, not on the property.